selenium/codenvy-selenium-test/src/main/java/com/codenvy/selenium/core/client/OnpremTestOrganizationServiceClientForAdmin.java

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) [2012] - [2017] Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ *******************************************************************************/
+package com.codenvy.selenium.core.client;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import org.eclipse.che.selenium.core.provider.TestApiEndpointUrlProvider;
+import org.eclipse.che.selenium.core.requestfactory.TestHttpJsonRequestFactoryForAdmin;
+
+/**
+ * This util is handling the requests to Organization API.
+ */
+@Singleton
```

Comment:
If we have several implementations it is recommended to use `Named` annotation for binding.